include/SDL3/SDL_time.h

```diff
@@ -0,0 +1,205 @@
+/*
+Simple DirectMedia Layer
+Copyright (C) 1997-2024 Sam Lantinga <slouken@libsdl.org>
+
+This software is provided 'as-is', without any express or implied
+warranty.  In no event will the authors be held liable for any damages
+arising from the use of this software.
+
+Permission is granted to anyone to use this software for any purpose,
+including commercial applications, and to alter it and redistribute it
+freely, subject to the following restrictions:
+
+1. The origin of this software must not be misrepresented; you must not
+   claim that you wrote the original software. If you use this software
+   in a product, an acknowledgment in the product documentation would be
+   appreciated but is not required.
+2. Altered source versions must be plainly marked as such, and must not be
+   misrepresented as being the original software.
+3. This notice may not be removed or altered from any source distribution.
+*/
+
+#ifndef SDL_time_h_
+#define SDL_time_h_
+
+/**
+ *  \file SDL_time.h
+ *
+ *  Header for the SDL realtime clock and date/time routines.
+ */
+
+#include <SDL3/SDL_error.h>
+#include <SDL3/SDL_stdinc.h>
+
+#include <SDL3/SDL_begin_code.h>
+/* Set up for C function definitions, even when using C++ */
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ *
+ * A structure holding a calendar date and time broken down into its components.
+ */
+typedef struct SDL_DateTime
+{
+    int year;                  /**< Year */
+    int month;                 /**< Month [01-12] */
+    int day;                   /**< Day of the month [01-31] */
+    int hour;                  /**< Hour [0-23] */
+    int minute;                /**< Minute [0-59] */
+    int second;                /**< Seconds [0-60] */
+    int nanosecond;            /**< Nanoseconds [0-999999999] */
+    int day_of_week;           /**< Day of the week [0-6] (0 being Sunday) */
+    int utc_offset;            /**< Seconds east of UTC */
+} SDL_DateTime;
+
+/**
+ *
+ * The preferred date format of the current system locale.
+ *
+ * \sa SDL_GetDateTimeLocalePreferences
```

Comment:
This function doesn't exist in the API anymore. Should it? (also see below for SDL_TIME_FORMAT)

Reply:
I would say that setting the properties at init time is adequate, as locale info is generally something that's only configured once when setting up a new system or device, and then never (or very rarely) changed afterward.

Reply:
Done.